cypher-shell/src/integration-test/java/org/neo4j/shell/commands/CypherShellIntegrationTest.java

```diff
@@ -139,26 +136,28 @@ public void resetInTxScenario() throws CommandException {
         shell.execute("MATCH (n) RETURN n");
 
         ArgumentCaptor<String> captor = ArgumentCaptor.forClass(String.class);
-        verify(logger, times(3)).printOut(captor.capture());
+        verify(logger, times(2)).printOut(captor.capture());
 
         List<String> result = captor.getAllValues();
-        assertThat(result.get(2), is("n\n(:TestPerson {name: \"Jane Smith\"})"));
+        assertThat(result.get(1), is("n\n(:TestPerson {name: \"Jane Smith\"})"));
     }
 
     @Test
     public void commitScenario() throws CommandException {
         beginCommand.execute("");
         shell.execute("CREATE (:TestPerson {name: \"Joe Smith\"})");
+        shell.execute("CREATE (:TestPerson {name: \"Jane Smith\"})");
+        shell.execute("MATCH (n:TestPerson) RETURN n");
```

Comment:
The order being different on your machine compared to linux annoys me. Let's change the query so it always returns the same order:

```
MATCH (n:TestPerson) RETURN n ORDER BY n.name
```